app.js

```diff
@@ -1,11 +1,17 @@
 const express = require('express')
 const app = express()
-
 app.use(express.json())
-const { getAllTopics } = require('./controller')
+const { getAllTopics, getApi} = require('./controller')
 
 app.get("/api/topics", getAllTopics)
 
+app.get('/api', getApi)
```

Comment:
You have two endpoints setup here doing different things but on the same route - lets only have the one set of behaviours for each endpoint.